spec/requests/unauthorized_api_access_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe 'POST /api/v1/booking_requests' do
+  scenario 'unauthorized access' do
+    given_a_permitted_gds_sso_user
```

Comment:
This would ultimately be a `User` that has a specific role, to permit API access. I agree the language isn't particularly clear though. I'll change it.